staging/src/k8s.io/sample-controller/OWNERS

```diff
@@ -2,10 +2,10 @@
 
 approvers:
   - sttts
-  - munnerz
 reviewers:
   - sttts
-  - munnerz
```

Comment:
I'm happy to be removed from here if needs be, but if you take a look at my contributions and recent PRs to this area of the codebase, I've been reviewing, approving and dealing with PRs to it.

https://github.com/kubernetes/kubernetes/pull/106530
https://github.com/kubernetes/kubernetes/pull/105719
https://github.com/kubernetes/kubernetes/pull/105247
https://github.com/kubernetes/kubernetes/pull/103281
https://github.com/kubernetes/kubernetes/pull/101871

This is a pretty low-volume component given its nature (and I've not been contributing a tonne in the core k/k repository aside from this kind of maintenance work, hence my volume of contributions is low).

I can really appreciate the efforts here to improve quality of approvers/reviewers though - but I'm happy to (and have done) continue to review & approve changes to this area of the codebase if that help is still wanted :)

Reply:
@munnerz thanks for confirming that you are still active. reverting this change!